manifests/modsecurity-configmap.yaml

```diff
@@ -0,0 +1,18 @@
+filebeat:
+  config:
+    modules:
+      path: /usr/share/filebeat/modules.d/*.yml
+      reload:
+        enabled: true
+  processors:
+    - dissect:
+        tokenizer: "%{message}"
+        target_prefix: ""
+        ignore_failure: true
+  output:
+    {{- with index .filebeat 0 }}
+    elasticsearch:
+      hosts: ["{{ .scheme }}://{{ .url }}:{{ .port }}"]
+      user: {{ .user }}
+      password: {{ .password }}
```

Comment:
I think we we wouldn want to push to a dedicated index so that a different index lifecycle policy can be used

Reply:
Ok, so have a section in the karina config for the elasticsearch details for modsecurity?